examples/two_windows.rs

```diff
@@ -24,40 +24,49 @@ fn main() {
         .add_system(ui_second_window_system);
 
     let render_app = app.sub_app_mut(RenderApp);
-    let mut graph = render_app.world.get_resource_mut::<RenderGraph>().unwrap();
 
-    bevy_egui::setup_pipeline(
-        &mut graph,
-        bevy_egui::RenderGraphConfig {
-            window_id: *SECOND_WINDOW_ID,
-            egui_pass: SECONDARY_EGUI_PASS,
-        },
-    );
+    render_app.add_system_to_schedule(ExtractSchedule, init_second_window);
```

Comment:
this method no longer exists in bevy main. see: https://github.com/DGriffin91/bevy_egui/pull/4

Reply:
Oops I replied in the wrong place, was meaning to add it  the line in lib.rs. regardless, that linked PR makes it build with latest main.